lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,69 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+                if ($('.accordion').length) {
+                    navigation.openAccordion();
+                }
+            },
+
+            openAccordion: function() {
+                navigation.checkForCurrent();
+                navigation.listenForClick();
+            },
+
+            checkForCurrent: function() {
+                var active = $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') ? $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') : 0,
+                    activeSection = $('.accordion .button-chapter:eq(' + active + ')');
+
+                navigation.closeAccordions();
+                navigation.openAccordionSection(activeSection);
+            },
+
+            listenForClick: function() {
+                $('.accordion').on('click', '.button-chapter', function(event) {
+                    navigation.closeAccordions(event.currentTarget);
+                    navigation.openAccordionSection(event.currentTarget);
```

Comment:
In both of these functions you are creating a jQuery object from the element so it would make sense to create it only once in this function and pass it to the close and open functions.